skeletonkey/core.py

```diff
@@ -9,9 +9,11 @@
     load_yaml_config,
     add_args_from_dict,
     add_yaml_extension,
-    namespace_to_nested_namespace,
+    key_to_nested_key,
+    Key
 )
 
+TARGET_KEYWORD: str = "_target_"

```

Comment:
I like these changes, but I feel like this should be a separate PR.

Reply:
I agree. I intended to make them as part of a separate PR, but I was accidentally working on the wrong branch of my repo when I added the new class and didn't think it was worth it to try to detangle the changes to make them separate PRs. In the future, I'd make sure they were separate.